functional-tests/.env

```diff
@@ -1,5 +1,6 @@
-BASEURL=http://127.0.0.1:8000/
+BASE_URL=http://127.0.0.1:8000/gwells/
 GWELLS_USERNAME=bdduser1
 GWELLS_VIEWER_USERNAME=bdd-viewer
 GWELLS_SUBMISSION_USERNAME=bdd-submission
-GWELLS_REGISTRY_USERNAME=bdd-registry
\ No newline at end of file
+GWELLS_REGISTRY_USERNAME=bdd-registry
+GWELLS_PASSWORD=
```

Comment:
I added a .env file, and a bit of code in build.gradle to read it and set the environment variables.